neurodamus/connection.py

```diff
@@ -265,8 +266,9 @@ def add_synapse_configuration(self, configuration):
         if configuration is not None:
             self._configurations.append(configuration)
 
-    def override_mod(self, mod_override):
-        assert mod_override.exists("ModOverride"), "ModOverride requires hoc config obj"
+    def set_mod_override(self, mod_override):
+        """Set a valid mod override"""
+        assert mod_override is not None, "ModOverride cannot be None"
```

Comment:
For readability, I would prefer to write a setter and getter with `property` decorator. 
And do you think it possible to have some unit tests for the `ConnectionBase` class? You can add a test for the getter and setter of modoverride in this PR.

Reply:
how do you write a setter with the property decorator? @property functions cannot have inputs except self and setters have always at least 1 input apart from self...

Reply:
ok, after discussion I got it. @property for the getter and @mod_override.setter for the setter. Done